FileSystem/FileIOTests.js

```diff
@@ -35,11 +39,11 @@ function file_io_tests() {
     FileIO.openFile('PRINT_TEST', fileNumber, OpenMode.INPUT);
     var content = FileIO.openFiles[fileNumber].content;
     var actualContent =
-      'This is a test\r\n\r\nZone 1        Zone 2\r\nHello World\r\n     5 leading spaces \r\n         Hello\r\nFalse is a Boolean value\r\n12-02-1969  is a date\r\nNull is a null value\r\nError 32767 is an error value\r\n';
+        'This is a test\r\n\r\nZone 1        Zone 2\r\nHello World\r\n     5 leading spaces \r\n         Hello\r\nFalse is a Boolean value\r\n12-02-1969  is a date\r\nNull is a null value\r\nError 32767 is an error value\r\n';
```

Comment:
Can we make this statement more readable? Split across lines?

Reply:
According to this thread - https://stackoverflow.com/questions/508269/how-do-i-break-a-string-across-more-than-one-line-of-code-in-javascript, one way of splitting a string across lines is using `\`
So I split the string like this 

```
 var actualContent =
        'This is a test\r\n\r\nZone 1        Zone 2\
        \r\nHello World\r\n     5 leading spaces \
        \r\n         Hello\
        \r\nFalse is a Boolean value\
        \r\n12-02-1969  is a date\
        \r\nNull is a null value\
        \r\nError 32767 is an error value\
        \r\n';
```

But it didn't work and the below image shows it to be exactly equal! but it still says test fails. So this method doesn't work. 

![image](https://user-images.githubusercontent.com/66052573/85979322-db941180-b9fd-11ea-8c5a-7d28c9e92123.png)

Reply:
Okay, splitting the string itself worked - 
```
'This is a test\r\n\r\n' +
        'Zone 1        Zone 2\r\n' +
        'Hello World\r\n' +
        '     5 leading spaces \r\n' +
        '         Hello\r\n' +
        'False is a Boolean value\r\n' +
        '12-02-1969  is a date\r\n' +
        'Null is a null value\r\n' +
        'Error 32767 is an error value\r\n';
```

I will refactor all tests to this method.